src/BinaryParsers/VersionConstants.cs

```diff
@@ -4,9 +4,9 @@ namespace Microsoft.CodeAnalysis.IL
 {
     public static class VersionConstants
```

Comment:
>VersionConstants [](http://example.com/codeflow?start=24&length=16)

I found this file, looks like we should add in the Release-Process md file

Reply:
@shaopeng-gh Agree. Please update the release process.

Reply:
updated